app/javascript/packs/provider-dialogs-common.js

```diff
@@ -53,9 +53,10 @@ function reactModal(buttonData) {
   };
 
   const Component = ManageIQ.component.getReact(buttonData.component_name);
+  const modalId = buttonData.component_name.split(/(?=[A-Z])/).join('-').toLowerCase();
   const inner = () => <Component {...props} />;
 
-  renderModal(__(buttonData.modal_title), inner);
+  renderModal(__(buttonData.modal_title), modalId, inner);
```

Comment:
What about the other `renderModal` call?

Since you're changing the meaning of the second param, this breaks it, right?